core/pom.xml

```
@@ -43,6 +43,7 @@ THE SOFTWARE.
     <spring.version>2.5.6.SEC03</spring.version>
     <groovy.version>2.4.12</groovy.version>
     <hamcrest.version>2.2</hamcrest.version>
+    <junit.jupiter.version>5.5.2</junit.jupiter.version>
```

Any particular reason you've chosen 5.5.2 instead of the 5.6.2 release from April 2020?
```suggestion
    <junit.jupiter.version>5.6.2</junit.jupiter.version>
```

---

I just copied the version from the cli. I plan to move the version to the parent-pom or the bom.